moveit_ros/moveit_servo/src/collision_monitor.cpp

```diff
@@ -50,6 +50,13 @@ CollisionMonitor::CollisionMonitor(const planning_scene_monitor::PlanningSceneMo
   , planning_scene_monitor_(planning_scene_monitor)
   , collision_velocity_scale_(collision_velocity_scale)
 {
+  scene_collision_request_.distance = true;
+  scene_collision_request_.contacts = true;
```

Comment:
I believe `contacts` is optional unless you want to print the colliding pair (please test, though)

```suggestion
  scene_collision_request_.contacts = false;
```

Reply:
it is optional (it actually works without) -- I put it there because it was also on in Humble, but I can get rid of it if you prefer.

Reply:
I will say it will be useful if we ever try to implement something that uses the contact pairs to determine whether the velocity is towards/away from collision... but not needed for now.